.github/heroku-postgres/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    heroku = {
+      source = "heroku/heroku"
+      version = "~> 4.0"
+    }
+  }
+}
+
+//resource "heroku_app" "default" {
```

Comment:
Remove commented-out code?

Reply:
Oh actually this was commented because the Heroku account running tests doesn't have permission to delete an App. If an idle Heroku App is generating unnecessary bills, we could promote the account and uncomment to drop the App.